default.nix

```diff
@@ -137,6 +136,25 @@ let
   deb-packages = moveDerivations "deb-packages" (map buildDeb packageDescs);
   rpm-packages = moveDerivations "rpm-packages" (map buildRpm packageDescs);
 
+  releaseFile = writeTextFile {
+    name = "release-notes.md";
+    text = ''
+      Automatic release on ${builtins.substring 0 8 timestamp}
+
+      This release contains assets based on [${babylonnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${babylonnet.rev}) of babylonnet branch and
+      [${mainnet.rev} revision](https://gitlab.com/tezos/tezos/tree/${mainnet.rev}) of mainnet branch from [tezos repository](https://gitlab.com/tezos/tezos/).
+      <!--
+      When making a new release, replace `autorelease` with actual release tag:
```

Comment:
Oh, sorry, the tag turned out to be `auto-release` (with a dash). Please change here and below (in the link).